tools/skbstat_example.txt

```diff
@@ -0,0 +1,88 @@
+Demonstrations of skbstat, the Linux BPF/bcc version.
+
+
+skbstat prints detailed information about the packet (skb) processed by the kernel,
+including ingress/egress/drop. we can pass filter parameters to it, such as "-p tcp -P 80":
+
+# skbstat.py -p tcp -P 80 --input --output
+TIME     NUM    FUNC/STACK
+20:04:29        [INGRESS]
+         1      tcp_v4_rcv
+         1      ip_local_deliver_finish
+         1      ip_local_deliver
+         1      ip_route_input_noref
+         1      ip_rcv_finish
+         1      ip_rcv
+         1      netif_receive_skb_internal
```

Comment:
If you can have an example which does not have all 1 numbers (as in the above) and have some explanation, that will be much better.

Reply:
I'll fix this.